starter-project-web/test/rebecca.spec.js

```diff
@@ -0,0 +1,57 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils'
+import Vuex from 'vuex'
+import Vuetify from 'vuetify'
+import Rebecca from '../pages/Rebecca'
+
+const localVue = createLocalVue()
+localVue.use(Vuex)
+localVue.use(Vuetify)
+
+describe('Rebecca Page', () => {
+  let actions
+  let store
+  let state
+  let vuetify
+
+  beforeEach(() => {
+    state = {
+      posts: [{ id: 1, title: 'post', body: 'post' }]
+    }
+
+    actions = {
+      getPosts: jest.fn()
+    }
+
+    store = new Vuex.Store({
+      modules: {
+        rebecca: {
+          state,
+          actions,
+          namespaced: true
+        }
+      }
+    })
+    vuetify = new Vuetify()
+  })
+
+  it('should renders posts', async () => {
+    const wrapper = shallowMount(Rebecca, { store, localVue, vuetify })
```

Comment:
You can use beforeEach function to mount the component, rather than repeatedly mounting in each test cases